src/test/scala/mesosphere/marathon/api/akkahttp/PathMatchersTest.scala

```diff
@@ -67,4 +67,31 @@ class PathMatchersTest extends UnitTest with GroupCreation with ScalatestRouteTe
       AppPathIdLike(Path("/restart")) shouldBe Unmatched
     }
   }
+
+  "PodsPathIdLike matcher" should {
```

Comment:
Property based tests! Yay!

Reply:
Hehe, yeah. We could actually use random strings. That's be fun :).